ffbs-debugbathosts/files/usr/sbin/update-bat-hosts

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/lua
```

Comment:
Might be nice to have this in `ffbs-debugbathosts/luasrc` instead of `ffbs-debugbathosts/files` so that the Lua compressor can reduce the size of the script.